frontends/web/src/routes/buy/moonpay-terms.tsx

```diff
@@ -94,7 +94,7 @@ export const MoonpayTerms = ({ account, onAgreedTerms }: TProps) => {
         <p>{t('buy.info.disclaimer.protection.description', { name })}</p>
         <p>
           <A className={style.link} href="https://www.moonpay.com/privacy_policy">
-            {t('buy.info.disclaimer.privacyPolicy')}
+            {t('buy.info.disclaimer.moonpayPrivacyPolicy')}
```

Comment:
aha, related to my other comment.. since you rename anyway maybe make it consistent with the key of the pocket link?

Reply:
Restored the previous name to avoid modifying keys